### ISSUE TITLE
Fails when passing multiple lines

### ISSUE THREAD

**Issue by hanspagel** [stackblitz-labs/pkg.pr.new#117]
Hey, I’m trying to run the following command:

```yaml
      - name: Publish on Stackblitz
        run: npx pkg-pr-new publish \
          ./packages/api-client \
          ./packages/api-reference \
          ./packages/api-reference-editor \
          ./packages/api-reference-react \
          ./packages/build-tooling \
          ./packages/cli \
          ./packages/code-highlight \
          ./packages/components \
          ./packages/docusaurus \
          ./packages/draggable \
          ./packages/echo-server \
          ./packages/express-api-reference \
          ./packages/fastify-api-reference \
          ./packages/galaxy \
          ./packages/hono-api-reference \
          ./packages/mock-server \
          ./packages/nestjs-api-reference \
          ./packages/nextjs-api-reference \
          ./packages/nuxt \
          ./packages/oas-utils \
          ./packages/object-utils \
          ./packages/play-button \
          ./packages/themes \
          ./packages/use-codemirror \
          ./packages/use-toasts \
          ./packages/use-tooltip \
          ./packages/void-server
```

And I think that syntax is fine (might be wrong, though), but the CLI seems to not trim the spaces or something. Anyway, this is the error I get:

```diff
+Error:  ENOENT: no such file or directory, open '/home/runner/work/scalar/scalar/ ./packages/api-client/package.json'
  at async open (node:internal/fs/promises:639:25)
  at async Object.readFile (node:internal/fs/promises:1249:14)
  at async writeDeps (/home/runner/.npm/_npx/e[8](https://github.com/scalar/scalar/actions/runs/9713468178/job/26810327808?pr=2310#step:12:9)f433c388b218fc/node_modules/pkg-pr-new/dist/index.js:3278:1[9](https://github.com/scalar/scalar/actions/runs/9713468178/job/26810327808?pr=2310#step:12:10))
  at async Object.run (/home/runner/.npm/_npx/e8f433c388b218fc/node_modules/pkg-pr-new/dist/index.js:3202:32)
  at async runCommand (/home/runner/.npm/_npx/e8f433c388b218fc/node_modules/pkg-pr-new/dist/index.js:2196:16)
  at async runCommand (/home/runner/.npm/_npx/e8f433c388b218fc/node_modules/pkg-pr-new/dist/index.js:2187:[11](https://github.com/scalar/scalar/actions/runs/9713468178/job/26810327808?pr=2310#step:12:12))
  at async runMain (/home/runner/.npm/_npx/e8f433c388b218fc/node_modules/pkg-pr-new/dist/index.js:2323:7)
```

Shouldn’t be `'/home/runner/work/scalar/scalar/ ./packages/api-client/package.json'`, but `'/home/runner/work/scalar/scalar/packages/api-client/package.json'`, I guess.

**Comment by Aslemammad** [comment]
Hey! Thank you so much for reporting, I'm taking a look!

**Comment by Aslemammad** [comment]
Just released pkg-pr-new@0.0.10
